client/tests/src/main/scalajs/sttp/tapir/client/tests/ClientContext.scala

```diff
@@ -0,0 +1,14 @@
+package sttp.tapir.client.tests
+
+import cats.effect.unsafe.IORuntime
+import scala.concurrent.ExecutionContext
+
+object ClientContext {
```

Comment:
Maybe `ClientPlatform` would be a better name? I guess it's more standard and better reflects that we're dealing with platform-specific code

Reply:
Renamed ClientContext to ClientTestsPlatform in latest commit.
https://github.com/softwaremill/tapir/pull/2519/commits/c39f2f15c325ddbd89ed278d9f1199eafa936b17